go-controller/pkg/ovn/gress_policy.go

```diff
@@ -23,11 +23,10 @@ type gressPolicy struct {
 	// IP addresess.
 	peerAddressSet AddressSet
 
-	// nsAddressSets holds the names of all namespace address sets
-	nsAddressSets sets.String
-
-	// sortedPeerAddressSets has the sorted peerAddressSets
-	sortedPeerAddressSets []string
+	// peerV4AddressSets  has Address sets  for all name spaces and pod selectors for IPv4
+	peerV4AddressSets sets.String
+	// peerV4AddressSets  has Address sets  for all name spaces and pod selectors for IPv6
+	peerV6AddressSets sets.String
```

Comment:
second comment should say "V6" not "V4". Also, there are double spaces in two places in each comment, and "namespaces" should be a single word

Reply:
Done